src/sentry/receivers/features.py

```diff
@@ -303,24 +297,19 @@ def record_inbound_filter_toggled(project, **kwargs):
 @alert_rule_created.connect(weak=False)
 def record_alert_rule_created(
     user,
-    project,
-    rule,
-    rule_type,
-    is_api_token,
+    project: Project,
+    rule_id: int,
```

Comment:
🎉 nice. will this apply to all the invocations you have for `alert_rule_created`?

Reply:
yup I switched all of them to use id